tests/integration/testdata/sync/Python/events/event.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Are we using this file in the integration tests?

Reply:
Nope, this was generated with sam init, I will remove this